### PR TITLE
don't use backdrop for dialogs

### PR DESCRIPTION
**Pull request by mawinter69**
The backdrop can make dialogs look really bad when there are large areas with dark color right behind the dialog (see screenshot).

Before:
image

After: 
image

### Testing done
Manual testing

### Proposed changelog entries

- Prevent backdrop color from affecting dialogs that appear above the backdrop.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by StefanSpieker — APPROVED** [review]
Thanks for the fix, I also noticed that today.

File: war/src/main/scss/components/_dialogs.scss
Comment (janfaracik) [inline war/src/main/scss/components/_dialogs.scss:8]: Could we potentially keep the backdrop filter but remove the `brightness()` (which needs to be removed anyway as it's breaking on HDR displays...):

```css
  background-color: color-mix(in srgb, var(--background) 85%, transparent);
  box-shadow: var(--dialog-box-shadow), 0 0 0 100vmax rgba(23, 28, 37, 0.6);
  backdrop-filter: blur(30px);
  animation: jenkins-dialog-animate-in 0.25s cubic-bezier(0, 0.68, 0.5, 1.5);
  overflow: hidden;
  padding: $jenkins-dialog-padding 0 0 0;
  display: flex;
  flex-direction: column;
  gap: $jenkins-dialog-padding;

  &__title {
    font-size: 1.125rem;
    font-weight: 500;
    padding: 0 $jenkins-dialog-padding;
    color: var(--text-color);
  }
```

Something like the above - the blur is still there just drastically toned down (so we still get a little colour bleed, but nothing overtly distracting like it is now). `::backdrop` has been killed in favour of the increased `box-shadow` as backdrop appears behind the dialog, darkening it, whereas box shadow masks as expected.

Comment (mawinter69) [reply war/src/main/scss/components/_dialogs.scss:8]: I would prefer that the dialog has a clear one color only background. It should not be influenced by what is behind the dialog in any way.
btw the same applies to drop down menus and tooltips
image
It just looks not good I think

**Pull request merged**
